tests/suites/test_suite_alignment.function

```diff
@@ -303,35 +303,29 @@ void unaligned_access_endian_aware(int size, int offset, int big_endian)
     }
     /* Overwrite sentinel with endian-aware write macro */
     if (big_endian) {
-        switch (size) {
-            case 16:
-                MBEDTLS_PUT_UINT16_BE(read, x, offset);
-                break;
-            case 24:
-                MBEDTLS_PUT_UINT24_BE(read, x, offset);
-                break;
-            case 32:
-                MBEDTLS_PUT_UINT32_BE(read, x, offset);
-                break;
-            case 64:
-                MBEDTLS_PUT_UINT64_BE(read, x, offset);
-                break;
-        }
+        /* Done this way rather than with a switch() to help ensure they
```

Comment:
What's “they”?

I guess it refers to the `MBEDTLS_PUT_xxx_` macros, but that's from the context of the pull request. It's not clear within the context of the code.

Also, “this way” refers not only to using if/else chains, but also not having braces.

Validating the macros like this is a good idea, but I'd prefer to do it separately from their functional testing, and the code needs to make it clear what's going on.

PoC:
```
#define CHECK_STATEMENT_MACRO_CALL(statement) \
    do { \
        if (0) statement; \
        else (void) "If there's a syntax error here, the argument passed to CHECK_STATEMENT_MACRO_CALL is missing do { ... } while (0)"; \
    } while (0)

CHECK_STATEMENT_MACRO_CALL(MBEDTLS_PUT_UINT16_BE(read, x, offset));
```